lib/commands/submit_order/process_piped.js

```diff
@@ -0,0 +1,58 @@
+'use strict'
+
+const { Order } = require('bfx-api-node-models')
+const orderTypeParam = require('../../params/order_type')
+const marketParam = require('../../params/market')
+const amountParam = require('../../params/amount')
+const priceParam = require('../../params/price')
+
+/**
+  * Converts raw parameters from a piped source (or raw user input) to valid
+  * values, and validates them. Supports all order types.
+  *
+  * @param {Object} args
+  * @param {boolean} requireConfirmation
+  * @return {Object} processed
+  */
+module.exports = (args, requireConfirmation) => {
+  const data = {
+    symbol: marketParam('market', args.symbol),
+    amount: amountParam('amount', +args.amount),
+    type: orderTypeParam('order type', args.type)
+  }
+
+  if (!args.affiliateCode) {
+    data.affiliateCode = 'xZvWHMNR'
```

Comment:
default code?

Reply:
Yes, it's the internal HF tracker; forgot to add configuration for a custom code, will open a new PR for it.

Reply:
ok